normandy/recipes/filters.py

```diff
@@ -393,6 +394,54 @@ def capabilities(self):
         return {"jexl.transform.date"}
 
 
+class ProfileCreateDateFilter(BaseFilter):
+    """
+        This filter is meant to distinguish between new and existing users.
+        Target users who have a profile creation date older than or newer than
+        a given date. This date must be specified in days since January 1st 1970.
+
+        .. attribute:: type
+
+            ``profile_creation_date``
+
+        .. attribute:: direction
+
+           :example: ``newer_than``
+
+       .. attribute:: date
+
+          :example: ``18657``
```

Comment:
Sorry for not seeing this before. I'm looking at this with fresh eyes, and it seems like this should probably be a date. And in fact it seems that's how you are using it in the code? The docs don't seem to match the code.

Reply:
Yea.. I think I just hurriedly added this docstring after the fact without thinking about it enough, sorry about that.